composer.json

```diff
@@ -50,9 +50,9 @@
         "name": "z3/randomizer",
         "version": "31.0.11",
         "source": {
-          "url": "https://github.com/KatDevsGames/z3randomizer",
+          "url": "https://github.com/cassidoxa/z3randomizer",
           "type": "git",
-          "reference": "2a3d6fae5824f4a3bdcc7c23ff8a3a123686ce08"
```

Comment:
Is this not merged into Kat's branch yet?

Reply:
It is, I'll change that real quick